plugin/src/main/groovy/com/novoda/gradle/command/Input.groovy

```diff
@@ -4,7 +4,23 @@ import org.gradle.api.tasks.TaskAction
 
 class Input extends AdbTask {
 
-    Closure script
+    InputSpec inputSpec
+
+    /**
+     * Manual creation of Input task is deprecated.
+     * Please refer to scripting documentation for details:
+     * https://github.com/novoda/gradle-android-command-plugin#input-scripting
+     */
+    @Deprecated
+    void setScript(Closure script) {
```

Comment:
Looks like a new method deprecated but it really is the existing property being converted to `write-only` property and deprecated. Here is what I want to achieve. 

Existing implementations will mark it as deprecated. 


![screen shot 2018-01-08 at 11 25 27 am](https://user-images.githubusercontent.com/763339/34680151-79c6117e-f498-11e7-91ae-efb51ff18731.png)